src/metadata.ts

```diff
@@ -24,6 +24,8 @@ export interface AppMetadata extends Metadata {
    */
   readonly description: string
 
+  readonly main?: string
```

Comment:
I think we should not add such option. User should use standard `package.json` main. And, so, our check should check such field in the package.json.